### DISCUSSION TITLE
Forced update frequency

### DISCUSSION THREAD

**Discussion by sideup66** [gtxaspec/wz_mini_hacks#382] [General]
Does anyone know how often wyze will try to update device firmware if they decide to force the current version? Will it cause constant update attempts that fail if block device updates is on in wz_mini hacks causing constant camera reboots? Or will it try maybe once or twice in the day?

**Comment by virmaior** [comment]
Ever since the big rollout in February, I haven't really encountered many forced upgrade attempts. I think I heard all of the cameras reboot once about a month ago and I assume that was such an attempt.

**Comment by mrlt8** [comment]
I believe `https://api.wyzecam.com/app/v2/upgrade/get_upgrade_firmware` should show if the firmware `is_force_upgrade`

**Comment by sideup66** [comment]
Ok, so it may only try once or twice and not continuously? I believe another way they do forced updates is by having the wyze app become incompatible with cameras under a specific firmware version. This is obviously a non issue if a user is on RTSP, however I do not use it. I have been hesitant to do the upgrade because frankly the setup I'm using is In a good place. It does everything I need it to do and keeps the camera reliable. I suppose I will if I'm forced but when he changelog lists "security improvements" I'm not sure what they did outside of adding the encrypted wifi password....

**Reply by virmaior** [reply]
Right now,  I actually prefer a modified version of `/multicam.html` . It's only 1fps but it can show me 6 cameras at once and allow me to pick which one is biggest on the screen. The lower fps is not super important for our main application (seeing a flying squirrel in the dark) and the ability to rapidly switch between cameras is way better than suffering wyze app's authentication process. 

Things I still need to figure out in order to not need the app:
1. irled control from multicam.html - quite doable as `cmd irled` already works
2. make the largest camera HLS - much harder as I haven't even gotten that to work for one camera yet.
3. have it tell me if something is amiss with writing to the SD card.

it's on one of my to-do lists, but this is one of *several* different hobbies and most of my time is when my wife is asleep -- which is a time when I can't really have the camera off or move it around.

**Reply by sideup66** [reply]
Right but if I understand that would still pull a stream from wyze rtsp streams for the different cameras, right? If I'm understanding anything NON rtsp OR camera cmd will still work. I'm sure rtsp will be figured out for the new firmware eventually....
